src/main/kotlin/no/nb/mlt/wls/infrastructure/synq/SynqOrderPayload.kt

```diff
@@ -53,15 +52,15 @@ data class ShippingAddress(
 
 fun Order.toSynqPayload() =
     SynqOrderPayload(
-        orderId = hostOrderId,
+        orderId = hostName.toString().uppercase() + "_" + hostOrderId,
```

Comment:
Nooooooo... not the underscore... we hates the undersore, yes yes, we hates it... Golum! Golum!